mmaction/integration/nncf/utils.py

```diff
@@ -0,0 +1,99 @@
+import torch
+from collections import OrderedDict
+from contextlib import contextmanager
+
+try:
+    import nncf
+    _is_nncf_enabled = True
+except ImportError:
+    _is_nncf_enabled = False
+except RuntimeError as _e:
+    _is_nncf_enabled = False
+    print('Attention: RuntimeError happened when tried to import nncf')
+    print('           The reason may be in absent CUDA devices')
+    print('           RuntimeError:')
+    print('           ' + str(_e), flush=True)
+
+
+
+def is_nncf_enabled():
+    return _is_nncf_enabled
+
+
+def check_nncf_is_enabled():
+    if not is_nncf_enabled():
+        raise RuntimeError('Tried to use NNCF, but NNCF is not installed')
+
+
+def get_nncf_version():
+    if not is_nncf_enabled():
+        return None
+    return nncf.__version__
+
+
+if is_nncf_enabled():
+    try:
+        from nncf.torch.checkpoint_loading import load_state
+        from nncf.torch.dynamic_graph.context import get_current_context
+        from nncf.torch.dynamic_graph.context import \
+            no_nncf_trace as original_no_nncf_trace
+    except ImportError:
+        raise RuntimeError(
```

Comment:
Do we really need to raise an exception hear instead of processing it in lazy-mode during the direct call?